omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptResource1_9Test.java

```diff
@@ -135,6 +143,35 @@ public void testSetNames() throws Exception {
     }
 
     @Test
+    public void testSetMappings() throws Exception {

```

Comment:
This test doesn't seem to be what I expected it to be like, please add a test to ConceptController1_9Test and do a post kind of like [here](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ConceptController1_8Test.java#L316)